src/PhpPact/Consumer/Listener/PactTestListener.php

```diff
@@ -101,29 +76,27 @@ public function endTestSuite(TestSuite $suite): void
             } elseif (!($tag = \getenv('PACT_CONSUMER_TAG'))) {
                 print 'PACT_CONSUMER_TAG environment variable was not set. Skipping PACT file upload.';
             } else {
```

Comment:
I don't think tag is required for `pact-broker publish` command.

Running this command:

```
bin/pact-ruby-standalone/bin/pact-broker publish
```

I will get:

```
No value provided for required options '--consumer-app-version', '--broker-base-url'
```

Reply:
See https://github.com/pact-foundation/pact-php/pull/298

Reply:
Resolve to make code reviewing easier. Feel free to reopen or add comment.